bin/chandra_repro

```diff
@@ -266,12 +266,16 @@ def update_aspsol_to_aspsolobi(asol_list,mydir, params):
     asp_offaxis_corr = make_tool("asp_offaxis_corr")
     asp_offaxis_corr(outfile, "acis")  # instrume doesn't matter
     
-    dmhedit.infile = outfile
-    dmhedit.file = ""
-    dmhedit.operation = "add"
-    dmhedit.key = "CONTENT"
-    dmhedit.value = "ASPSOLOBI"
-    dmhedit()
+    # Update asol header with updated content and NOM values
+    from pycrates import read_file, set_key, get_keyval
+    tab = read_file(outfile, mode="rw")
+    set_key(tab, "CONTENT", "ASPSOLOBI")
+    for key in ['RA', 'Dec', 'Roll']:
+        val = get_keyval(tab, "{}_PNT".format(key))
```

Comment:
Now we're free of Python 3.5 we can use the easier-to-read

    f"{key}_PNT"

Reply:
I find mixing old/new syntax creates a Frankenstein's monster  -- prefer to keep consistent.